sonar-php-plugin/src/test/resources/org/sonar/plugins/php/phpunit/sensor/phpunit.systemout.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@kinow could you provide a shorter test file? It's hard to find `system-out` here :)